src/com/pablo/utils/ConvertidorUnidades.java

```diff
@@ -0,0 +1,88 @@
+package com.pablo.utils;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class ConvertidorUnidades {
+	static Scanner scan = new Scanner(System.in);
```

Comment:
Como la variable Scan no debería ser utilizada fuera de esta clase podrías ENCAPSULARLA.

Agregando el modificador de acceso prívate. 

private static final Scanner scan = new Scanner(System.in);

El private encapsularía la variable
static hace que solamente exista una instancia y final hace que nadie pueda sobreescribir volviéndola una constante, ya que nadie debería poder sobreescribirla.

Reply:
Done